docs/connect/odbc/using-azure-active-directory.md

```diff
@@ -79,7 +79,7 @@ It's possible to use Microsoft Entra authentication options when creating or edi
 
 `Authentication=ActiveDirectoryMsi` for Microsoft Entra managed identity authentication
 
-![The DSN creation and editing screen with Managed Service Identity authentication selected.](windows/create-dsn-ad-msi.png)
+![The DSN creation and editing screen with Managed Service Identity authentication selected.](windows/create-dsn-ad-sys-msi.png)

```

Comment:
I would revert this change and simply update the existing screenshot with one like this:

![image](https://github.com/MicrosoftDocs/sql-docs/assets/26262303/c160756d-43c7-4cde-b0f0-b9ca52ed4082)